package.json

```diff
@@ -172,9 +172,7 @@
       "js",
       "jsx"
     ],
-    "moduleDirectories": [
-      "node_modules"
-    ],
+    "transformIgnorePatterns": ["/node_modules/(?!(query-string|decode-uri-component|split-on-first|filter-obj)/)"],
```

Comment:
This is ugly, we should try to spend some time moving to esm in the project. I tried this during 30min but this was quite some work... I wonder if moving to esm would be a useful first step for moving to typescript, or rather a distaction.